src/SamplesApp/SamplesApp.Skia.Tizen/App.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Tizen.Applications;
+using ElmSharp;
+using SkiaSharp.Views.Tizen;
+using SkiaSharp;
+using WUX = Windows.UI.Xaml;
+using Uno.UI.Runtime.Skia;
+using System.Diagnostics;
+
+namespace SamplesApp.Skia.Tizen
+{
+	class MainClass
+	{
+		static void Main(string[] args)
+		{
+			Elementary.Initialize();
+			Elementary.ThemeOverlay();
```

Comment:
We should move this to the `TizenHost` class, so that normal apps can benefit from it.